Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -248,6 +251,22 @@ class PlaceCarouselViewController: UIViewController {
 
     fileprivate func updatePlaces(forLocation location: CLLocation) {
         self.placesProvider.updatePlaces(forLocation: location)
+        let placeMonitoringRadius = RemoteConfigKeys.getDouble(forKey: RemoteConfigKeys.searchRadiusInKm) / 2
```

Comment:
Rather than `searchRadius / 2`, should we just make the place monitoring radius its own remote config value?